### PR TITLE
feature/86cuwqpa7-Create-Booking-Details-page-secondary

### PR DESCRIPTION
**Pull request by gericW**
- Implementing Booking Details page
- Missing function for accept and reject button

### CODE REVIEW

**Review by RyanAHicks — COMMENTED** [review]
Reviewed

**Review by RyanAHicks — APPROVED** [review]
Approved.

**Review by kbib0002 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts :)

File: package-lock.json
Comment (nethbtju) [inline package-lock.json:1]: Why is there two package-lock,json?

**Pull request closed without merging**